uint8 id
uint8 color
geometry_msgs/PoseStamped poseOrigin
geometry_msgs/PoseStamped poseFcu